pom.xml

```diff
@@ -205,7 +205,7 @@
     If you are changing Arrow version specification, please check ./python/pyspark/sql/utils.py,
```

Comment:
Here too `python/pyspark/sql/pandas/utils.py` :-). BTW was this comment helpful when you upgrade @BryanCutler? If it was more bothering than helpful, feel free to remove.

Reply:
I think I didn't read that one, since I've done this many times :)  It would probably be helpful to someone new, so I'll update it and keep it.